### PR TITLE
fix slow boot

### PR DESCRIPTION
**Pull request by rafapaezbas**
This PR fixes the slow boot issue introduced by https://github.com/holepunchto/pear/commit/91a048a2cb1b32de3774da7f32eda609cc6ef2dd and https://github.com/holepunchto/pear/commit/f4b2c01bd1f6bcab959fd0a8d6811e66d318df7a, notice that adding the list of links in package.json as trusted apps is removed. Instead a `--trusted` flag is included for every worker executed.

### CODE REVIEW

File: subsystems/sidecar/index.js [hunk 1]
```diff
@@ -824,14 +786,17 @@ class Sidecar extends ReadyResource {
...
-    await this.permit({ key: state.key, encryptionKey }, client)
-
     try {
       await appBundle.calibrate()
-    } catch (err) {
-      await session.close()
-      throw err
+    } catch (error) {
+      if (error.code === 'DECODING_ERROR') {
+        const err = ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
```
Comment (davidmarkclements) [inline subsystems/sidecar/index.js:793]:
```suggestion
        const err = new ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
```

File: subsystems/sidecar/index.js [hunk 2]
```diff
@@ -756,13 +725,15 @@ class Sidecar extends ReadyResource {
...
-    if (trusted.has(z32) === false) {
-      const err = ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
-      app.report({ err })
-      return { startId, bail: err }
+    if (!trustedFlag) {
+      const aliases = Object.values(ALIASES).map(hypercoreid.encode)
+      const trusted = new Set([...aliases, ...((await permits.get('trusted')) || [])])
+      const z32 = hypercoreid.encode(state.key)
+      if (trusted.has(z32) === false) {
+        const err = ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
```
Comment (davidmarkclements) [inline subsystems/sidecar/index.js:733]:
```suggestion
        const err = new ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
```

File: subsystems/sidecar/index.js [hunk 3]
```diff
@@ -693,6 +661,7 @@ class Sidecar extends ReadyResource {
     const id = client.userData?.id || `${client.id}@${startId}`
     const app = client.userData = client.userData?.id ? client.userData : new this.App({ id, startId, session })
     const state = new State({ id, env, link, dir, cwd, flags, args, cmdArgs, run: true })
+    const trustedFlag = cmdArgs.some((arg) => arg === '--trusted')
```
Comment (davidmarkclements) [inline subsystems/sidecar/index.js:664]: not needed - `flags.trusted`

Comment (rafapaezbas) [reply subsystems/sidecar/index.js]: updated, I was setting the flag in the wrong place

**Pull request merged**